cluster/manifests/node-monitor/daemonset.yaml

```diff
@@ -73,6 +73,10 @@ spec:
               protocol: TCP
         - image: registry.opensource.zalan.do/teapot/prometheus-node-exporter:v1.2.2
           args:
+{{- if eq .Cluster.ConfigItems.node_exporter_experimental_metrics "true" }}
```

Comment:
You need to add a default, otherwise it'll crash I think.

Reply:
Oops, done
Thanks!